application/controllers/handler.js

```diff
@@ -49,10 +49,24 @@ module.exports = {
       const fileType = '.jpeg';
       const filePath = path.join(conf.fsPath, 'slideThumbnails/' + fileName + fileType);
       const html = request.payload;
+      let document = cheerio.load(html);
+      let pptxwidth = document('div[class=pptx2html]').css().width.replace('px', '');
+      let pptxheight = document('div[class=pptx2html]').css().height.replace('px', '');
+      pptxwidth = pptxwidth ? pptxwidth : 0;
+      pptxheight = pptxheight ? pptxheight : 0;
+      let width = 1024;
+      let height = 768;
+      if (pptxwidth !== 0 && pptxheight !== 0) {
+        width = pptxwidth;
+        height = pptxheight;
+      } else {
+        width = 'all';
```

Comment:
Okay. Just to get it right (I haven't tested it yet): WindowSize now defaults to 1024x768 (see docs), you might render a slide that is larger (e.g. wider) than this window and take a shot of either all the window content (this will almost never happen, as most slides contain height and width information) or the actual slide size (that might be larger than the window). Does it result in complete shot of the slide or is it cropped at the window border? E.g. the Slide is 1900x500. Will the shot result in 1900x500 or in 1024x500 or in 1024x768 (and is everything displayed that is supposed to be there?)